docs/testplan/VxLAN_BGP_EVPN_test_plan.md

```diff
@@ -0,0 +1,225 @@
+# Functionality and Scaling of BGP EVPN VxLAN
+
+- [Functional and Scale testing of BGP EVPN VxLAN](#functional-and-scale-testing-of-bgp-evpn-vxlan)
+  - [Overview](#overview)
+    - [Scope](#scope)
+    - [Keysight Testbed](#keysight-testbed)
+  - [Topology](#topology)
+    - [SONiC switch as Leaf](#sonic-switch-as-leaf)
+  - [Setup configuration](#setup-configuration)
+  - [Bgp Evpn VxLan HLD](#bgp-evpn-vxlan-hld)
+  - [Scalability Requirements](#scalability-requirements)
+  - [Test cases](#test-cases)

```

Comment:
Can following test cases be added. 
1) Interface flap with traffic on (access and uplink port) to make sure mac learning and tunnel comes up fine after interface up. 
2) Mac flush to make sure MAC converge after flush without any issues.